src/anet.c

```diff
@@ -381,19 +381,53 @@ static int anetSetReuseAddr(char *err, int fd) {
     return ANET_OK;
 }
 
-static int anetCreateSocket(char *err, int domain) {
+#define ANET_SOCKET_CLOEXEC 1
+#define ANET_SOCKET_NONBLOCK 2
+#define ANET_SOCKET_REUSEADDR 4
+static int anetCreateSocket(char *err, int domain, int type, int protocol, int flags) {
     int s;
-    if ((s = socket(domain, SOCK_STREAM, 0)) == -1) {
+
+    /* In general, SOCK_CLOEXEC won't have noticeable effect
```

Comment:
This comment doesn't really help me understand the context of the SOCK_CLOEXEC flag; I'm not really sure a comment is needed here at all. In both cases a flag is being set on the socket() call if possible, or being applied to the socket afterwards if not.

Reply:
I'm not sure I get your point here.

Reply:
I'm OK with that.